### ISSUE TITLE
No response triggered on iOS 8.1. App Extension

### ISSUE THREAD

**Issue by tomvlk** [Alamofire/Alamofire#869]
Using Alamofire with Cocoapods in an app extension (Today widget) on iOS 8.1 doesn't trigger the response handlers.
On iOS 9.0 it's all fine.

Versions of Alamofire tried: 2.0 and 3.0.
Request type: .GET, With HTTP Digest Authentication over HTTPS, Custom parameters URL encoding.
Response type: JSON

I validated on the server, and the request is still being executed. And even the authentication is going on. But still no response callback. In the app itself it's all working, also on iOS 8.1.

Should be anything with memory leaks? Because the today widget will use 14GB memory after 10 minutes running.

Details running software:
Xcode 7
iOS Target 8.1

Part of code for request call:
```swift
// Define some credentials here
let username = "username"
let password = "password"

var parameters:[String: AnyObject] = [:]
parameters["d"] = (UIDevice.currentDevice().identifierForVendor)!.UUIDString
parameters["v"] = "1.1.1"
parameters["b"] = "14414234"
parameters["uc"] = "1"

Alamofire.request(Alamofire.Method.GET, "https://somenicelink/", parameters: parameters, encoding: ParameterEncoding.URL)
    .authenticate(usingCredential: NSURLCredential(user: username, password: password, persistence: NSURLCredentialPersistence.ForSession))
    .responseJSON { response in
        let result = response.result
        print(result)
}
```

Thanks,
Tom

**Comment by cnoon** [comment]
Hi @tomvlk, would you be able to submit us a sample project that exposes the issue? I can't think of any reason why you would see this behavior. Is the request timing out?

**Comment by tomvlk** [comment]
Hi @cnoon. Currently I can't reproduce it myself. And it looks like it has something related with the memory leak and CPU lockup I got in the simulator.
Tried to disable the whole code. Use instruments to get the leaks. But it doesn't seem to be in my code or app as I can't get the leak address lead back to xcode lines.

I think it isn't Alamofire related. But more like an underlying API that will cause CPU lockup/memory leaking that will cause Alamofire to lose the request in some way.

I will continue to try and disable some stuff related to the widget. I can't test on a real device. I don't have iOS 8.1 on any devices anymore unfortunately.
